common/source/docs/common-advanced-configuration.rst

```diff
@@ -83,6 +83,7 @@ tuning options for the vehicle.
 [site wiki="copter"]
     Motor Thrust Scaling <motor-thrust-scaling>
 [/site]
+    Moving Vehicle Initialization <common-moving-vehicle-initialization>
```

Comment:
can we add "(aka Boat Mode)" here and on the page itself?

I wonder if "Moving Vehicle Initialisation" is clear enough.  How about "Starting from a moving platform"?... or "Initialising from a Moving Platform"?

Reply:
if someone searched they would find boat mode and be re-directed....but will change to "Initializing from a Moving Platform"